lib/compat.js

```diff
@@ -0,0 +1,136 @@
+const through = require('through2');
```

Comment:
How is this different from the lib/compat in dynastar?
https://github.secureserver.net/opensource/dynastar/blob/master/index.js